addons/website_twitter/static/tests/tours/twitter_scroller_reload_button.js

```diff
@@ -0,0 +1,44 @@
+odoo.define('website_twitter.tour.twitter_scroller_reload_button', function (require) {
+'use strict';
+
+var tour = require('web_tour.tour');
+var base = require('web_editor.base');
+
+tour.register('twitter_scroller_reload_button', {
+    test: true,
+    url: '/',
+    wait_for: base.ready(),
+}, [{
+    trigger: 'a[data-action=edit]',
+    content: 'Click the Edit button.',
+    extra_trigger: '.homepage',
+}, {
+    trigger: '#snippet_feature .oe_snippet:has(span:contains("Twitter Scroller")) .oe_snippet_thumbnail',
+    content: 'Drag the Twitter Scroller snippet and drop it in your page.',
+    run: 'drag_and_drop #wrap',
+}, {
+    trigger: '#wrap .twitter',
+    content: 'Activate the Twitter Scroller snippet editor.',
+}, {
+    trigger: '#wrap .twitter .twitter_timeline center',
+    content: 'Check if the reload button was added to the DOM.',
+    run: function () {
+        if ($('.btn span:contains("Reload")').length === 0) {
+            console.error('The twitter scroller reload button was not added to the DOM');
+        }
+    },
+}, {
+    trigger: '.oe_snippet_remove:last',
+    content: 'Remove the twitter scroller snippet.',
+},
+{
+    trigger: '#wrap',
+    content: 'Check if the reload button was removed from the DOM.',
```

Comment:
Couldn't we insert that test in the drag & drop snippet instead of spawning a new tour/browser/routing map which is very perf killer?

Reply:
The twitter scroller snippet is in a separate module, so I don't really see how?

Reply:
That's not problematic regarding the tour I was thinking about, but that tour is actually not in 13.0, I introduced it in 15.0.

This one: https://github.com/odoo/odoo/blame/ac417f28470a27b929a9c0999ee75e30c1777912/addons/website/static/tests/tours/snippets_all_drag_and_drop.js#L45

I guess we can merge the tour as it is in 13.0 and 14.0 and remove it + make it become just a specific step in the drag_and_drop test in 15.0.

Reply:
That tour is run post install, so it is including the website_twitter snippet, as you can see here http://runbot49.odoo.com/runbot/static/build/12892734-15-0/logs/test_post.txt grep `test_03_snippets_all_drag_and_drop`

Not very usual and best practice to use a website_xxx module stuff in website, but for this particular case it would be quite optimal :+1: 

Anyway, we will see in 15.0

Reply:
Yes, that does seem quite counter intuitive to me...